v19.1/cost-based-optimizer.md

```diff
@@ -67,10 +67,20 @@ The cost-based optimizer can often find more performant query plans if it has ac
 
 For best query performance, most users should leave automatic statistics enabled with the default settings. The information provided in this section is useful for troubleshooting or performance tuning by advanced users.
 
-To control how often the automatic statistics jobs run on your cluster, adjust the following [cluster settings](cluster-settings.html). They define the target number of rows in a table that should be stale before statistics on that table are refreshed.
+#### Controlling statistics refresh rate
 
-- `sql.stats.automatic_collection.fraction_stale_rows`
-- `sql.stats.automatic_collection.min_stale_rows`
+Statistics are refreshed in the following cases:
+
+1. When there are no statistics.
+2. When it's been a long time since the last refresh, where "long time" is defined according to a moving average of the time across the last several refreshes.
+3. After each mutation operation ([`INSERT`](insert.html), [`UPDATE`](update.html), or [`DELETE`](delete.html)), the probability of a refresh is calculated using a formula that takes the [cluster settings](cluster-settings.html) shown below as inputs. These settings define the target number of rows in a table that should be stale before statistics on that table are refreshed.
+
+| Setting                                              | Details                                                                              |
+|------------------------------------------------------+--------------------------------------------------------------------------------------|
+| `sql.stats.automatic_collection.fraction_stale_rows` | Target fraction of stale rows per table that will trigger a statistics refresh       |
```

Comment:
Do we want to include the default values here? Seems like it would be useful, but there's the downside that they could become stale. Not sure what the general practice is for cluster settings.

Reply:
Our general practice for cluster settings (thus far) is to try really hard to point users to the (generated, up-to-date) cluster settings page, unless we really need to mention the settings.  Since we need them here in this case, I'd rather not include the default values here for the reason you said: staleness / the risk of the values being incorrect.